applications/pelion_client/README.rst

```diff
@@ -0,0 +1,522 @@
+
+.. include:: /includes/spm.txt
+
+Pelion Device Management requirements
+=====================================
+
+You need a developer account on the `Pelion Device Management Portal`_.
+
+Configuration
+*************
+
+The Pelion library is enabled with :option:`CONFIG_PELION_CLIENT` configuration option.
+
+For the library to work, you must enable and properly configure the Mbed TLS library.
+To connect to the Device Management, the device must be provisioned with valid credentials.
+For the firmware update procedure to work, you must provision the device with valid update resources.
+
+Mbed TLS
+========
+
+The Pelion library is depending on a properly configured Mbed TLS library.
+To simplify the development within the |NCS|, a predefined set of Mbed TLS configuration options was prepared.
+Set compatible with :ref:`nrfxlib:nrf_security` can be enabled using :option:`CONFIG_PELION_NRF_SECURITY` configuration option.
+
+Pelion credentials
+==================
+
+To be able to connect to the Device Management, you need to complete the following configuration:
+
+1. Generate and download a developer certificate file from the `Pelion Device Management Portal`_.
+   This file contains keys used for securing connection with the Pelion network.
+#. Copy the downloaded file to :file:`applications/nrf_desktop/configuration/common/`, replacing the default :file:`mbed_cloud_dev_credentials.c` file.
+#. The file is used according to the developer flow provisioning (see :ref:`pelion_client_device_provisioning`).
+   If the certificate is not present, :c:func:`fcc_developer_flow()` copies it from the developer certificate file (which is now part of the application binary).
+
+For details steps, see the `Provisioning development devices`_ guide in the Pelion documentation.
+
+Firmware update
+===============
+
+You can provide the devices with the new version of the firmware image over-the-air from the Pelion Device Management Portal (see :ref:`pelion_client_dfu`).
+
+Before the update, you must enable a compatible bootloader.
+The Pelion library is compatible with MCUboot.
+Enable it with the :option:`CONFIG_BOOTLOADER_MCUBOOT` Kconfig option.
+Along with the bootloader, you must also enable an image manager with the :option:`CONFIG_IMG_MANAGER` and :option:`CONFIG_MCUBOOT_IMG_MANAGER` Kconfig options.
+
+Additionally, if the application image is large, you may need to store the update image on an external flash device.
+In such case, enable the external flash support and correctly configure the partition layout.
+
+The Pelion library's update manager is enabled when you select the :option:`CONFIG_PELION_UPDATE` Kconfig option.
+This option enables components required for image transport and storage.
+For the update campaign to be recognized and the image be accepted, you need to provision the device with the valid update resources.
+To simplify the development process, you can have the update resources created by the update manifest generation tool.
```

Comment:
I suggest adding a direct link to the manifest creation tool, similar to the link above to the development credentials. The link should be https://developer.pelion.com/docs/device-management/v4.7/updating-firmware/manifest-tool.html

Reply:
Added